src/Controllers/QuestionTemplateController.php

```diff
@@ -82,6 +139,12 @@ public function showQuestionDetailsEdit(Request $request, array $requestAttribut
         return $this->renderer->renderView("admin-question-details.phtml", $params);
     }
 
+    /**
```

Comment:
"Displays the 'Create Question' form"?
(Not sure that's what the form is actually called.)

Can you avoid the workaround with empty params? E.g. ensure the variables are set in the template?